models/ed/R/veg2model.ED2.R

```diff
@@ -0,0 +1,234 @@
+#' Writes ED specific IC files
+#'
+#' @param obs data either passed from upstram or NULL
+#' @param inputinfo
+#' @param runinfo run from settings
+#' @param outfolder where to write files
+#' @param host 
+#' @param con bety$con
+#' @param overwrite
+#' @return ic.id initial conditions file input id
+#' @export
+#' @author Istem Fer
+#'  We need the in.path, the out folder, the start and endates and the lat/long. All of these are in setting$run. 
+veg2model.ED2 <- function(obs = NULL, inputinfo, runinfo, outfolder, host, con, overwrite = FALSE){
+  
+  start_year <- lubridate::year(runinfo$start.date)
+  end_year   <- lubridate::year(runinfo$end.date)
+  startdate   <- lubridate::as_date(paste0(start_year, "-01-01"))
+  enddate     <- lubridate::as_date(paste0(end_year, "-12-31"))
+  lat <- as.numeric(runinfo$site$lat)
+  lon <- as.numeric(runinfo$site$lon)
+  
+  # handle file details
+  if(inputinfo$output == "site"){
+    
+    formatname <- "ED2.site"
+    dbfilename <- "site.file"
+    fileprefix <- paste("siteid", runinfo$site$id, inputinfo$source, start_year, end_year,
```

Comment:
there is this small `site.style = FALSE / TRUE` thing we talked about, it's same for css/pss but slightly more rounded lat/lon for the site file, if that naming difference is unnecessary I can mode it out of the if chunk